src/bindings/swig/lua/kdb.i

```diff
@@ -85,8 +85,10 @@
 }
 
 %extend kdb::Key {
-  Key(const char *keyName) {
-    return new kdb::Key(keyName, KEY_END);
+  Key(const char *name, uint64_t flags = 0) {
```

Comment:
No. Swig doesn't understand kdb_unsigned_long_long_t or any other cheerless kdb types.
However swig will map the type to the supported platform types on it's own.